arrayWithRandomValues.java

```diff
@@ -0,0 +1,69 @@
+public class arrayWithRandomValues {

```

Comment:
Class name should always start with upper case

Reply:
Read java code convention https://www.oracle.com/technetwork/java/codeconventions-150003.pdf